c_glib/test/parquet/test-arrow-file-writer.rb

```diff
@@ -89,10 +89,10 @@ def test_write_table
   def test_write_chunked_array
     schema = build_schema("enabled" => :boolean)
     writer = Parquet::ArrowFileWriter.new(schema, @file.path)
-    writer.new_row_group(2)
+    writer.new_row_group()
```

Comment:
@kou can we break here as well?

Reply:
Yes.

Could you omit `()` for no argument method call?

```suggestion
    writer.new_row_group
```